src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -27,6 +27,7 @@ export const ButtonGroup = ({size, isReversed, variant, color, className, childr
                             size={size}
                             variant={variant}
                             isReversed={isReversed}
+                            isDisabled={isDisabled}
```

Comment:
Si isDisabled overrides the button isDisabled ? I have cases where I need some buttons to be disabled , and some to be enabled.

Reply:
So I close this PR and we will manage the disabled state in each button